### DISCUSSION TITLE
Laurarium Road Access

### DISCUSSION THREAD

**Discussion by TheRealAyCe** [Keriew/augustus#896] [Q&A] [answered]
Manual says it needs road access. Is that true? What for?

**Comment by Ouaz** [comment]
Yes, the lararium needs road access within two tiles (the manual will be updated accordingly)
The description says _"Small sacrifices and prayers may be made here."_, so we can suppose that citizens need a road to access it. ^^

**Reply by TheRealAyCe** [reply]
And what happens if it doesn't have road access?

**Reply by UberWaffe** [reply, marked as answer]
Strictly speaking it doesn't need road access but engineer and prefect access, because they can collapse or catch fire.
Hence why within 2 tiles, since that is the range of the engineers and prefects.
